modules/react/avatar/lib/Avatar.tsx

```diff
@@ -38,7 +38,7 @@ export interface AvatarProps extends React.ButtonHTMLAttributes<HTMLButtonElemen
   as?: 'div';
   /**
    * The object-fit CSS property sets how the content of a replaced element,
-   * such as an <img> or <video>, should be resized to fit its container.
+   * such as an `<img>` or `<video>`, should be resized to fit its container.
```

Comment:
The canvas site interpreted these as elements, so I put them in ticks to avoid that

Reply:
This fix should go into a separate PR against the `support` branch since the PR which introduced the issue (#2337) was merged into `support`.